File: tutorials/vbnet/hello_world/hw_i.vb
Public NotInheritable Class hwConst

	Private Sub New()
	End Sub

	'defines for service HelloWorld
	Public Const sidHelloWorld As UInteger = (SocketProAdapter.BaseServiceID.sidReserved + 1)

	Public Const idSayHelloHelloWorld As UShort = (CUShort(SocketProAdapter.tagBaseRequestID.idReservedTwo) + 1)
	Public Const idSleepHelloWorld As UShort = (idSayHelloHelloWorld + 1)
	Public Const idEchoHelloWorld As UShort = (idSleepHelloWorld + 1)
End Class

File: tutorials/csharp/hello_world/hw_i.cs

public static class hwConst
{
    //defines for service HelloWorld
    public const uint sidHelloWorld = (SocketProAdapter.BaseServiceID.sidReserved + 1);

    public const ushort idSayHelloHelloWorld = ((ushort)SocketProAdapter.tagBaseRequestID.idReservedTwo + 1);
    public const ushort idSleepHelloWorld = (idSayHelloHelloWorld + 1);
    public const ushort idEchoHelloWorld = (idSleepHelloWorld + 1);
}